earthmover/operations/row.py

```diff
@@ -124,3 +124,51 @@ def execute(self, data: 'DataFrame', **kwargs):
                 raise
 
             return data.sort_values(by=self.columns_list, ascending=(not self.descending))
+
+
+class FlattenOperation(Operation):
+    """
+
+    """
+    allowed_configs: Tuple[str] = (
+        'operation', 'repartition', 
+        'flatten_column', 'left_wrapper', 'right_wrapper', 'separator', 'value_column', 'trim_whitespace'
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Only 'column' or 'columns' can be populated
+        self.flatten_column  = self.error_handler.assert_get_key(self.config, 'flatten_column', dtype=str, required=True)
+        self.left_wrapper = self.error_handler.assert_get_key(self.config, 'left_wrapper', dtype=str, required=False, default="[\"")
+        self.right_wrapper = self.error_handler.assert_get_key(self.config, 'right_wrapper', dtype=str, required=False, default="\"]")
+        self.separator = self.error_handler.assert_get_key(self.config, 'separator', dtype=str, required=False, default=',')
+        self.value_column = self.error_handler.assert_get_key(self.config, 'value_column', dtype=str, required=True)
+        self.trim_whitespace = self.error_handler.assert_get_key(self.config, 'trim_whitespace', dtype=str, required=False, default=" \t\r\n\"")
+
+
+    def execute(self, data: 'DataFrame', **kwargs) -> 'DataFrame':
+        """
+
+        :return:
+        """
+        super().execute(data, **kwargs)
+
+        # target_dtypes = data.head(1).rename(columns={self.flatten_column: self.value_column})
+        target_dtypes = data.dtypes.to_dict()
+        target_dtypes.update({self.value_column: target_dtypes[self.flatten_column]})
+        del target_dtypes[self.flatten_column]
+        data = data.map_partitions(self.flatten_partition, meta=target_dtypes)
+        return data
+
+    def flatten_partition(self, df):
```

Comment:
I worry that persisting three objects here will use more memory than just chaining operations on a single one.